keyboards/sofle_pico/info.json

```diff
@@ -0,0 +1,249 @@
+{
+    "manufacturer": "Ryan Neff",
+    "keyboard_name": "Sofle Pico",
+    "maintainer": "Ryan Neff <github@JellyTitan.com>",
+    "bootloader": "rp2040",
+    "bootloader_instructions": "Hold the BOOT button while plugging in the usb cable",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP14", "pin_b": "GP15"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "dynamic_tapping_term": true,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "oled": true,
+        "rgb_matrix": true,
+        "wpm": true
+    },
+    "matrix_pins": {
+        "cols": ["GP1", "GP2", "GP3", "GP4", "GP5", "GP8"],
+        "rows": ["GP9", "GP10", "GP11", "GP12", "GP13"]
+    },
+    "mouse_key": {
+        "delay": 10,
+        "enabled": true,
```

Comment:
```suggestion
```

Reply:
@waffle87 since the mouse_key is defaults to `false` - should those other settings be removed as well?